src/modules/olMap/handler/featureInfo/featureInfoItem.provider.js

```diff
@@ -43,7 +45,13 @@ export const getBvvFeatureInfo = (olFeature, layerProperties) => {
 		)?.getCoordinates() ?? [];
 	const getContent = () => {
 		const descContent = olFeature.get('description') || olFeature.get('desc');
-		const geometryContent = html`</div><ba-geometry-info .statistics=${stats}></ba-geometry-info><div class='chips__container'><ba-profile-chip .coordinates=${elevationProfileCoordinates}></ba-profile-chip>`;
+		const geometryContent = html`
+		</div>
+			<ba-geometry-info .statistics=${stats}></ba-geometry-info>
+			<div class='chips__container'>
+				<ba-profile-chip .coordinates=${elevationProfileCoordinates}></ba-profile-chip>
+				<ba-export-vector-data-chip .exportData=${new KML().writeFeatures([olFeature])}></ba-export-vector-data-chip>
```

Comment:
Coordinates of the KML will be in feature projection (3857, see ol docs), that's not what we want here